### DISCUSSION TITLE
New-FalconHostGroup : What are the allowed parameters for -AssignmentRule ?

### DISCUSSION THREAD

**Discussion by DimitrisBinichakis** [CrowdStrike/psfalcon#348] [Q&A]
Hello,

I've been playing around with New-FalconHostGroup, and so far I have only been able to create groups with assignment rules related to sensor/falcon grouping tags. When I try to add a parameter such as type or domain, I get a 500 error. 

@bk-cs , are you able to clarify on what are the acceptable parameters for "-AssignmentRule" ?

Related docs:
https://github.com/crowdstrike/psfalcon/wiki/New-FalconHostGroup

**Comment by bk-cs** [comment]
You can create groups in the console and view those using `Get-FalconHostGroup` to see what accepted properties look like.

Try `product_type_desc:'Workstation'` for something like type, or `domain:'abc123'` for domain.
